enterprise_access/apps/api/serializers/__init__.py

```diff
@@ -2,6 +2,7 @@
 API serializers module.
 """
 from .content_assignments.assignment import (
+    ContentMetadataForAssignmentSerializer,
```

Comment:
Adding this for local dev convenience, along with the `shell_plus` imports below.